RedLockNet.SERedis/RedLock.cs

```diff
@@ -14,707 +14,712 @@
+            var host = GetHost(cache.ConnectionMultiplexer);
+
+            RedLockInstanceResult result;
+
+            try
+            {
+                logger.LogTrace("LockInstanceAsync enter {host}: {redisKey}, {LockId}, {expiryTime}", host, redisKey, LockId, expiryTime);
+                var redisResult = await cache.ConnectionMultiplexer
+                    .GetDatabase(cache.RedisDatabase)
+                    .StringSetAsync(redisKey, LockId, expiryTime, When.NotExists, CommandFlags.DemandMaster)
+                    .ConfigureAwait(false);
+
+                result = redisResult ? RedLockInstanceResult.Success : RedLockInstanceResult.Conflicted;
+            }
+            catch (Exception ex)
+            {
+                logger.LogDebug("Error locking lock instance {host}: {exMessage}", host, ex.Message);
+
+                result = RedLockInstanceResult.Error;
+            }
+
+            logger.LogTrace("LockInstanceAsync exit {host}: {redisKey}, {LockId}, {result}", host, redisKey, LockId, result);
+
+            return result;
+        }
+
+        private RedLockInstanceResult ExtendInstance(RedisConnection cache)
+        {
+            var redisKey = GetRedisKey(cache.RedisKeyFormat, Resource);
+            var host = GetHost(cache.ConnectionMultiplexer);
+
+            RedLockInstanceResult result;
+
+            try
+            {
+                logger.LogTrace("ExtendInstance enter {host}: {redisKey}, {LockId}, {expiryTime}", host, redisKey, LockId, expiryTime);
+
+                // Returns 1 on success, 0 on failure setting expiry or key not existing, -1 if the key value didn't match
+                var extendResult = (long)cache.ConnectionMultiplexer
+                    .GetDatabase(cache.RedisDatabase)
+                    .ScriptEvaluate(ExtendIfMatchingValueScript, new RedisKey[] { redisKey }, new RedisValue[] { LockId, (long)expiryTime.TotalMilliseconds }, CommandFlags.DemandMaster);
+
+                result = extendResult == 1 ? RedLockInstanceResult.Success
+                    : extendResult == -1 ? RedLockInstanceResult.Conflicted
+                    : RedLockInstanceResult.Error;
+            }
+            catch (Exception ex)
+            {
+                logger.LogDebug("Error extending lock instance {host}: {exMessage}", host, ex.Message);
+
+                result = RedLockInstanceResult.Error;
+            }
+
+            logger.LogTrace("ExtendInstance exit {host}: {redisKey}, {LockId}, {result}", host, redisKey, LockId, result);
+
+            return result;
+        }
+
+        private void UnlockInstance(RedisConnection cache)
+        {
+            var redisKey = GetRedisKey(cache.RedisKeyFormat, Resource);
+            var host = GetHost(cache.ConnectionMultiplexer);
+
+            var result = false;
+
+            try
+            {
+                logger.LogTrace("UnlockInstance enter {host}: {redisKey}, {LockId}");
```

Comment:
Values to log are missing

Reply:
Please fix placeholder casing. Microsoft recommends to use `PascalCase` - should be `{Host}`, `{RedisKey}` and `{LockId}`